factcast-site/documentation-docsy/content/en/Concept/fact.md

```diff
@@ -84,7 +84,7 @@ When a fact is read from FactCast, it is guaranteed to have two field set in the
 |_ser|long / int64| unique serial number for the fact, that determines a before/after relationship between facts|
 |_ts |long / int64| timestamp in milliseconds, when this fact was published to factcast.|
 
-As you can see, all meta-data attributes prefixed with "_" are supposed to be server created, so please do not use an "_" prefix yourself.
+As you can see, all meta-data attributes prefixed with "\_" are supposed to be server created, so please do not use an "\_" prefix yourself.
```

Comment:
At least here: https://docs.factcast.org/concept/fact/ this leads to unintended formatting. Cheers!